splunk-otel-android/src/main/java/com/splunk/rum/ConnectionUtil.java

```diff
@@ -44,6 +44,14 @@
 
     void startMonitoring(Supplier<NetworkRequest> createNetworkMonitoringRequest, ConnectivityManager connectivityManager) {
         refreshNetworkStatus();
+        try {
+            registerNetworkCallbacks(createNetworkMonitoringRequest, connectivityManager);
+        } catch (Exception e) {
+            //if this fails, we'll go without network change events.
```

Comment:
Worth logging or making an event or anything?

Reply:
I'm honestly not sure. This is a rare issue, and it happens before the library is initialized, and I've never seen it locally..so...long story short, I don't know what might be useful, and where to put it.

Reply:
I added a warning log. Hopefully if anyone sees it, they will let us know